pkgs/applications/networking/modem-manager-gui/default.nix

```diff
@@ -18,22 +18,14 @@
 
 stdenv.mkDerivation rec {
   pname = "modem-manager-gui";
-  version = "0.0.19.1";
+  version = "0.0.20";
 
   src = fetchhg {
-    url = "https://linuxonly@bitbucket.org/linuxonly/modem-manager-gui";
-    rev = "version ${version}";
-    sha256 = "11iibh36567814h2bz41sa1072b86p1l13xyj670pwkh9k8kw8fd";
+    url = "http://hg.code.sf.net/p/modem-manager-gui/code";
```

Comment:
is there no https source?

Reply:
hmm, I guess from looking that seems to be the best place aside from a mirror

Reply:
Https works

Reply:
ahh awesome. when I tried it in my browser it asked for authentication.

Reply:
It actually doesn't
```
checking outputs of '/nix/store/iswv8scz05hnd68na1h6whxz5hinds5q-hg-archive.drv'...
getting https://hg.code.sf.net/p/modem-manager-gui/code (v0.0.20) into /nix/store/7k55mmphigcpayxwwcr6lw11ib4nb56p-hg-archive
warning: connection security to hg.code.sf.net is disabled per current settings; communication is susceptible to eavesdropping and tampering
abort: http authorization required for https://hg.code.sf.net/p/modem-manager-gui/code
error: --- Error --------------------------------------------------------------------------------------------------------------------------------------------------- nix-build
error: --- Error --- nix-daemon
builder for '/nix/store/iswv8scz05hnd68na1h6whxz5hinds5q-hg-archive.drv' failed with exit code 255; last 3 log lines:
  getting https://hg.code.sf.net/p/modem-manager-gui/code (v0.0.20) into /nix/store/7k55mmphigcpayxwwcr6lw11ib4nb56p-hg-archive
  warning: connection security to hg.code.sf.net is disabled per current settings; communication is susceptible to eavesdropping and tampering
  abort: http authorization required for https://hg.code.sf.net/p/modem-manager-gui/code
```

it probably worked for you because the archive was cached by nix. I had to do `nix-build -A modem-manager-gui.src`. Could you revert back?

Reply:
Wait, sorry, that was the version from the store. The authorization breaks the package.

Reply:
So, I could fetch it from another distro, like Debian (https://salsa.debian.org/debian/modem-manager-gui).

Reply:
That actually isn't uncommon in nixpkgs and would be acceptable. Go for it.